docs/05-technical-reference/00-custom-resources/telemetry-03-tracepipeline.md

```diff
@@ -2,6 +2,8 @@
 title: Telemetry - TracePipeline
 ---
 
+> **CAUTION:** Jaeger is [deprecated](https://kyma-project.io/blog/jaeger-deprecation) and is planned to be removed. If you want to install a custom Jaeger stack, take a look at [this tutorial](https://github.com/kyma-project/examples/tree/main/jaeger).
```

Comment:
Same is before: This is not related to Jaeger. Just change the endpoint in the example below.